composer.json

```diff
@@ -1,6 +1,25 @@
 {
+	"name": "phpbb-de/phpbb-translation",
+	"description": "German Translation of phpBB",
+	"type": "phpbb-language",
+	"homepage": "https://github.com/phpbb-de/phpbb-translation",
+	"version": "3.1.11-dev",
```

Comment:
Laut der Info: https://getcomposer.org/doc/04-schema.md kann man das weglassen, steht die Nummer ja auch noch in der Readme. Steht hier als "optional" bzw. entbehrlich, wenn man die Nummer auch von den tags herbekommt: https://getcomposer.org/doc/04-schema.md#version 

Habe mich aber an den .com-exts orientiert, die nutzen "version": noch.